.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    environment:
+      CC_TEST_REPORTER_ID: 0643ac526557dd490bc6f0542cac608a8815d7538dd580261e67dd334c07cc87
+      COVERAGE: true
+    docker:
+      # specify the version you desire here
+       - image: circleci/ruby:2.3.5-node-browsers
+         env:
+         - RAILS_ENV=test
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+       - image: redis:3.2
+
+    working_directory: ~/asis
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install Elasticsearch
+          command: |
+            curl -O https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.7.2.deb && sudo dpkg -i elasticsearch-1.7.2.deb && true
```

Comment:
Just curious.. why not version 1.7.3? I guess there are no significant differences between 1.7.2 and 1.7.3 w.r.t. asis.

Reply:
I used 1.7.2 because it's what we use for Travis: https://github.com/GSA/asis/blob/master/.travis.yml#L6

Reply:
Got it. Thanks for indulging my curiosity.

Reply:
NP. I'm sure I could use 1.7.3, but the CircleCI configs have been such a pain in the *** so far, that I tried to make as few deviations from our Travis setup, etc. as possible. I'm inclined to change it if/when we change our ES version for asis.